site/operational-guide/concepts.md

```diff
@@ -19,22 +19,23 @@ A peer is a running instance of Weave Net, typically one per host.
 
 ## Peer Name
 
-Peers in the weave network are identified by a 48-bit value formatted
-like an ethernet MAC address e.g. `01:23:45:67:89:ab`. This 'peer
+Peers on the Weave network are identified by a 48-bit value formatted
```

Comment:
hmm. I don't think either 'in' or 'on' is right here, though I haven't checked the rest of the docs for what terminology we are using there.